src/components/views/dialogs/InviteDialog.tsx

```diff
@@ -1536,9 +1536,9 @@ export default class InviteDialog extends React.PureComponent<Props, IInviteDial
             );
             dialogContent = (
                 <React.Fragment>
-                    <TabbedView
+                    <TabbedView<TabId>
```

Comment:
If you have to specify this explicitly and it isn't implied by the value of `activeTabId` then that suggests the types have broken down and need fixing.